src/main/resources/com/github/terma/jenkins/githubprcoveragestatus/Configuration/help-sonarToken.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Access token for Sonar instance
+</div>
```

Comment:
Thx a lot for help page 👍  Could you please specify here that ```sonarToken``` is optional.Additionally you can provide link on Sonar doc like https://docs.sonarqube.org/display/SONAR/User+Token